Task3/dashboard.js

```diff
@@ -0,0 +1,67 @@
+let info = document.querySelector(".card-body");
+let cityName;
+let weatherInfo = [];
```

Comment:
as you are assigning value as empty array inside function , you can just declare the variable here .